src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -536,23 +535,25 @@ protected void onMemberJoined(final XmppChatMember chatRoomMember)
      * Creates a new {@link ColibriConference} instance for use by this
      * {@link JitsiMeetConferenceImpl}.
      */
-    private ColibriConference createNewColibriConference(Jid bridgeJid)
+    private ColibriConferenceImpl createNewColibriConference(Jid bridgeJid)
```

Comment:
why does this need to return the sub-type?

Reply:
It doesn't, nice catch